### ISSUE TITLE
2FA no more needed?

### ISSUE THREAD

**Issue by vzlgdu** [twrecked/pyaarlo#64]
Hi, I've disabled 2fa for my account and it's still working...

It work web access, app access and python access...

but hadn't it become mandatory to use 2fa?

**Comment by m0urs** [comment]
Correct, in spite of Arlo's announcements , 2FA is still NOT mandatory ;-)

**Comment by pfmos** [comment]
I believe it is now, unless you know of a way to deactivate?
